### PR TITLE
Define `TensorMap` for CUDA 11

### PR DESCRIPTION
**Pull request by zasdfgbnm**

### CODE REVIEW

**Comment by zasdfgbnm** [comment]
!build

File: csrc/tma.cpp [hunk 1]
```diff
@@ -104,6 +104,12 @@ inline CUtensorMapFloatOOBfill getCUtensorMapFloatOOBfill(
 
 using TensorMap = CUtensorMap;
 
+#else
+
+struct TensorMap {
```
Comment (naoyam) [inline csrc/tma.cpp:109]: Is TMA supported with CUDA 11? Or is this to just make the build pass?

Comment (zasdfgbnm) [reply csrc/tma.cpp:109]: This is just to make the build pass.

Comment (naoyam) [reply csrc/tma.cpp:109]: OK, please leave a comment

**Review by naoyam — APPROVED** [review]
LGTM

File: csrc/tma.cpp (hunk 1 above)
Comment (zasdfgbnm) [reply csrc/tma.cpp:110]: added

**Pull request merged**